level_complete.gd

```diff
@@ -0,0 +1,8 @@
+extends Area2D
+export(String, FILE) var next_level = ""
+
+
+
+
+func _on_Area2D_body_entered(body):
+	get_tree().change_scene(next_level)
```

Comment:
Please rework to use functions provided by: https://github.com/Boxnet-Media/boxnetjam/pull/41

e.g. `global.next_level()`